README.md

```diff
@@ -148,6 +148,9 @@ PALOMA_KEYRING_PASS=<your Paloma key password>
 ETH_RPC_URL=<Your Ethereum mainnet RPC URL>
 ETH_PASSWORD=<Your ETH Key Password>
 ETH_SIGNING_KEY=<Your ETH SIGNING KEY>
+BNB_RPC_URL=<Your Binance mainnet RPC URL>
```

Comment:
We know what these two URLs should be, correct? We can just fill in the correct values here.

Reply:
not everybody would be using ours though. That's a paid endpoint, so not everybody who would be reading these instructions can use ours and folks also might have a preference for a specific provider